pkg/controller/certificates/internal/secretsmanager/secret.go

```diff
@@ -203,8 +204,22 @@ func (s *SecretsManager) setValues(crt *cmapi.Certificate, secret *corev1.Secret
 			delete(secret.Data, jksSecretKey)
 			delete(secret.Data, jksTruststoreKey)
 		}
-	}
 
+		// Add additional key format for the key
+		if crt.Spec.PrivateKey.AdditionalKeyFormats != nil {
+			for _, f := range crt.Spec.PrivateKey.AdditionalKeyFormats {
+				switch f.Type {
+				case cmapi.DERKeyFormat:
+					block, _ := pem.Decode(data.PrivateKey)
+					secret.Data["tls.der"] = block.Bytes
```

Comment:
suggestion: having thought about this a bit more, I think a different name for the secret key might be helpful.

`tls.der` has no hint that it's just the private key. From looking at the name, I might reasonably conclude that it contains a DER-encoded certificate chain. `tls.der` is arguably a better name than the already-existing k8s `tls.key` (which just provides no useful information about the format at all) but I think we can do better here.

Especially with private keys, I think it's important to be clear when something should be private. Including "key" or "privkey" or "private" somewhere in the name would help to reinforce that.

How about maybe `key.der` instead? I'd be open to other suggestions too.

Reply:
Good point !  `key.der` is a better name than `lts.der` . I'll make the changes.